commonv5/ts/helpers/request.ts

```diff
@@ -1,46 +1,56 @@
-import axios from "axios";
 import * as semver from "semver";
 import * as tl from "azure-pipelines-task-lib/task";
+import fetch from 'node-fetch'
 import Endpoint from "../sonarqube/Endpoint";
 
 interface RequestData {
   [x: string]: any;
 }
 
+const parseJson = async response => {
+  const text = await response.text()
+  try{
+    const json = JSON.parse(text)
+    return json
+  } catch(err) {
+    return text
+  }
```

Comment:
Would also be possible to check `response.headers['Content-Type']` to check if it's `application/json` or not. Although, this is probably in fact safer..